reactivesocket-examples/src/main/java/io/reactivesocket/examples/EchoClient.java

```diff
@@ -45,10 +46,10 @@ public static void main(String... args) throws Exception {
         TcpReactiveSocketConnector tcp =
             new TcpReactiveSocketConnector(new NioEventLoopGroup(8), setupPayload, System.err::println);
 
-        ReactiveSocket client = Builder.instance()
-            .withSource(source(address))
-            .withConnector(tcp)
-            .build();
+        ReactiveSocket client = ClientBuilder.instance()
+                                             .withSource(source(address))
+                                             .withConnector(tcp)
+                                             .build();
```

Comment:
After we are done with considerable cleanups, I think we should add a checkstyle file and re-format everything.